src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,62 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardOpenOption;
+
 public class WorkWithFile {
+    private static final String[] NAMES = {"supply", "buy", "result"};
+    private static final int INDEX_NAME = 0;
+    private static final int INDEX_AMOUNT_MONEY = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        File files = new File(toFileName);
+        int[] results = new int[3];
+        if (files.exists()) {
+            files.delete();
+        }
```

Comment:
You can avoid this logic if you write data into file using `writeString` method

Reply:
What that method means?